src/chrome/content/rules/Tampermonkey.xml

```diff
@@ -0,0 +1,30 @@
+<!--
+	For Tampermonkey.net
+
+		Works:
+
+			- www
+			- $
+			- forum
+			- safari
+
+		Doesn't Work:
+		nil
+-->
+<ruleset name="Tampermonkey.net">
+  <target host="tampermonkey.net" />
+  <target host="*.tampermonkey.net" />
+
+  <rule from="^http://tampermonkey\.net/"
+          to="https://tampermonkey.net/" />
+
+  <rule from="^http://(www|forum|safari)\.tampermonkey\.net/"
```

Comment:
Similar to my other comment, I would suggest breaking out each of these domains into its own target host and then using a simple http: -> https: redirect. I'm working on a style guide that I'll post soon.

So it would be like:
```
<target host="tampermonkey.net" />
<target host="www.tampermonkey.net" />
<target host="forum.tampermonkey.net" />
<target host="safari.tampermonkey.net" />
```

And again, this will allow you to drop the redundant test URL tags.